### PR TITLE
 move function DisableAllStages from protected to private

### PR DESCRIPTION
**Pull request by leike666666**
move function DisableAllStages from protected to private

### CODE REVIEW

**Review by rmlarsen — APPROVED** [review]

**Comment by leike666666** [comment]
@gbaned it has been stuck here for a long time. Please take a look at it.

**Comment by gbaned** [comment]
> @gbaned it has been stuck here for a long time. Please take a look at it.

@leike666666 Sorry for the slow response, it is waiting for internal approval. Thank you.

**Pull request merged**
